File: src/test/java/TC/세균증식Test.java
package TC;

import org.assertj.core.api.Assertions;
import org.example.TC.세균증식;
import org.junit.jupiter.api.Test;

public class 세균증식Test {

    private final 세균증식 germ = new 세균증식();

    @Test
    void solution(){
        int result = germ.solution(2, 10);
        int answer = 2048;
        Assertions.assertThat(result).isEqualTo(answer);

    }

}

File: src/test/java/TC(K)/세균증식Test.kt
package `TC(K)`

import io.kotest.core.spec.style.StringSpec
import org.assertj.core.api.Assertions
import org.example.TCK.세균증식


class 세균증식Test : StringSpec(

    {
        "세균증식 테스트 코드"{
            val n: Int = 2
            val t: Int =10
            val answer: Int = 2048
            val result:Int = 세균증식().solution2(n, t)
            Assertions.assertThat(result).isEqualTo(answer)
        }

        "세균증식 테스트 코드 2"{
            val n:Int = 7
            val t:Int = 15
            val answer: Int = 229376
            val result:Int = 세균증식().solution2(n,t)
            Assertions.assertThat(result).isEqualTo(answer)
        }

    }

)
